src/csv/output.txt

```diff
@@ -0,0 +1,5 @@
+{"book":"The Compound Effect","author":"Darren Hardy","amount":"5","price":"9.48"}
```

Comment:
The output does not meet the requirements in home task. `amount` field should be excluded in output JSON file

Reply:
fixed